### PR TITLE
User Manager Pagination

### PR DESCRIPTION
**Pull request by davidtanline**
add pagination and UI changes to User Manager

### CODE REVIEW

File: api/main_endpoints/routes/User.js [hunk 1]
```diff
@@ -142,6 +142,67 @@ router.post('/users', function(req, res) {
...
+router.get('/davidUsers', async (req, res) => {
+  // if (!checkIfTokenSent(req)) {
+  //   return res.sendStatus(FORBIDDEN);
+  // } else if (!checkIfTokenValid(req)) {
+  //   return res.sendStatus(UNAUTHORIZED);
+  // }
+  try {
+    const limit = parseInt(req.query.u);
+    const page = parseInt(req.query.page) - 1 || 0;
+    const search = req.query.search || '';
```
Comment (neelp03) [inline api/main_endpoints/routes/User.js:154]: Change name and add token check back in

File: populate.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  'Crunchy',
+  'Deafen',
+  'Earthy',
+  'Freedom',
+  'Geared',
+  'Humpty',
+  'Illenium',
+  'Janitor'
+];
+
```
Comment (neelp03) [inline populate.js:54]: We don't need this file anymore

File: src/Components/Routing/PrivateRoute.js [hunk 3]
```diff
@@ -1,29 +1,50 @@
-import React from 'react';
-import { Route, Redirect } from 'react-router-dom';
+import React, { Component } from 'react';
+// import { Route, Redirect } from 'react-router-dom';
```
Comment (neelp03) [inline src/Components/Routing/PrivateRoute.js:2]: uncomment all the stuff we need

File: src/Routing.js [hunk 4]
```diff
@@ -51,7 +53,7 @@ export default function Routing({ appProps }) {
     // new for Overview
     {
       Component: Overview,
-      path: '/user-manager',
+      path: '/old-user-manager',
       allowedIf: userIsOfficerOrAdmin,
       redirect: '/',
       inAdminNavbar: true
```
Comment (neelp03) [inline src/Routing.js:59]: ig we can delete this now

File: src/index.js [hunk 5]
```diff
@@ -1,13 +1,35 @@
 import React, { useEffect, useState } from 'react';
 import ReactDOM from 'react-dom';
-import { withRouter } from 'react-router-dom';
+// import { withRouter } from 'react-router-dom';
```
Comment (neelp03) [inline src/index.js:3]: do we need this?

File: src/Pages/Overview/Overview.js [hunk 6]
```diff
@@ -135,6 +135,7 @@ export default class OverviewBoard extends Component {
   render() {
     return (
```
Comment (neelp03) [inline src/Pages/Overview/Overview.js:136]: don't think we need this file anymore

**Review by neelp03 — CHANGES_REQUESTED** [review]

**Review by evanugarte — COMMENTED** [review]
haven't forgotten about this, will be merged soon (by summer)

**Comment by evanugarte** [comment]
closed by #1310

**Pull request closed without merging**